### PR TITLE
add max-variable-recurse parameter

### PR DESCRIPTION
**Pull request by msaf1980**
Add max-variable-recurse parameter for print depth of pointers, nested structs and etc 
See feature request from issue https://github.com/go-delve/delve/issues/1625

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -42,6 +42,9 @@ type Config struct {
 	// MaxArrayValues is the maximum number of array items that the commands
 	// print, locals, args and vars should read (in verbose mode).
 	MaxArrayValues *int `yaml:"max-array-values,omitempty"`
+	// MaxVariableRecurse is output evaluation depth of of nested struct members, array and
```
Comment (derekparker) [inline pkg/config/config.go]: nit: two "of"'s.

File: pkg/config/config.go [hunk 2]
```diff
@@ -216,6 +219,9 @@ substitute-path:
 # Maximum loaded string length.
 # max-string-len: 64
 
+# Output evaluation.
+# max-variable-recurse 1
```
Comment (derekparker) [inline pkg/config/config.go]: must have colon after setting, e.g.: `max-variable-recurse: 1`.

File: pkg/config/config.go (hunk 1 above)
Comment (msaf1980) [reply pkg/config/config.go]: Not understood, where double s ?

Comment (derekparker) [reply pkg/config/config.go]: No I mean, it says "of of" instead of just "of".

**Pull request merged**